ntbs-service/Migrations/20210408092956_AddReleaseVersion.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class AddReleaseVersion : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "ReleaseVersion",
+                columns: table => new
+                {
+                    Version = table.Column<string>(type: "nvarchar(450)", nullable: false),
```

Comment:
It doesn't really matter, but why did you choose 450 here?

Reply:
I didn't explicitly choose it, I think its the default for a string primary key.